<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>addRem</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:#FF7F50</string>
  </property>
  <widget class="QLabel" name="RegistrLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>381</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe Print</family>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Добавить участника</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="createLog_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>110</y>
     <width>321</width>
     <height>41</height>
    </rect>
   </property>
   <property name="toolTipDuration">
    <number>0</number>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:#ffa852</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="lineWidth">
    <number>0</number>
   </property>
   <property name="tabChangesFocus">
    <bool>false</bool>
   </property>
   <property name="documentTitle">
    <string/>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="overwriteMode">
    <bool>false</bool>
   </property>
   <property name="acceptRichText">
    <bool>true</bool>
   </property>
   <property name="placeholderText">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>180</y>
     <width>271</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="addFriendButt">
      <property name="styleSheet">
       <string notr="true"> display: inline-block;
    text-decoration: none;
    background-color: #ffa852;
    color: #006089;
    border: 3px solid #d9812f;
    border-radius: 5px;
    font-size: 16px;
    padding: 7px 10px; 
    transition: all 0.8s ease;</string>
      </property>
      <property name="text">
       <string>Добавить</string>
      </property>
      <property name="default">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="cancelButt">
      <property name="styleSheet">
       <string notr="true"> display: inline-block;
    text-decoration: none;
    background-color: #ffa852;
    color: #006089;
    border: 3px solid #d9812f;
    border-radius: 5px;
    font-size: 16px;
    padding: 7px 10px; 
    transition: all 0.8s ease;</string>
      </property>
      <property name="text">
       <string>Отмена</string>
      </property>
      <property name="default">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
